Source/ASDisplayNode.mm

```diff
@@ -2996,6 +2998,14 @@ - (void)willEnterHierarchy
   }
 }
 
+- (void)didEnterHierarchy {
+  ASDisplayNodeAssertMainThread();
+  ASDisplayNodeAssert(!_flags.isEnteringHierarchy, @"You should never call -willEnterHierarchy directly. Appearance is automatically managed by ASDisplayNode");
```

Comment:
Update. comment `You should never call -didEnterHierarchy`

Reply:
Done, thanks.